Update interfaces/web-nfc.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/52f31b9/whatwg/idl/web-nfc.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/563479802

lukebjerring reviewed (dismissed):
Interface changes should be tested (and come from https://github.com/w3c/web-nfc/commit/203b5ee0d8f1ee1066a6aa6a5d29845f050e0cca)

The interface deletions seem to be another case of https://github.com/tidoust/reffy-reports/issues/34

autofoolip committed:
Update interfaces/web-nfc.idl

Source: https://github.com/tidoust/reffy-reports/blob/52f31b9/whatwg/idl/web-nfc.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/563479802

foolip commented:
Changes look sensible now. There are some new failing tests, but no regression of existing tests on Chrome.
